File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    result = 0
    sorted_score = sorted(score, reverse=True)
    
    for i in range(0, len(score), m):
        if i + m <= len(score):
            result += min(sorted_score[i:i+m]) * m
            
    return result

File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    let result = 0;
    score.sort((a, b) => b - a);
    
    for (let i = 0; i < score.length; i += m) {
        if (i + m <= score.length) {
            result += Math.min(...score.slice(i, i + m)) * m;
        }
    }
    return result;
}